chore: remove obsolete pages and add prefix to `primevue` components

enzo-bitfly opened the pull request.

enzo-bitfly committed:
chore: remove obsolete `user settings` page

enzo-bitfly committed:
chore: remove obsolete `register` page

marcel-bitfly commented on frontend/utils/mobileAuth.ts:
**issue:** this `util` is used by `mobile app v2`

@manuelsc are you ok with us removing this?
You can always recover this via `git`

enzo-bitfly replied:
In our project it was only used in the `register` page, which we never used and I remove in this PR

enzo-bitfly committed:
chore: add prefix to `primevue` components

prevents primevue button from being resolved when using `<component :is="'button'"...`
